pages/linux/rename.md

```diff
@@ -1,6 +1,7 @@
 # rename
 
 > Renames multiple files.
+> This is the perl version of `rename`, which is the default on Debian. See `rename.ul` for the stand-alone version.
```

Comment:
Maybe make this more explicit? Also capitalize "Perl".

```suggestion
> This is the Perl version of `rename`, which is the default on Debian. See `rename.ul` for the util-linux version.
```

And perhaps to equalize the length of both lines, we could instead consider something like this:

> Rename multiple files using Perl regular expressions.
> For the C-based command included in util-linux, see `rename.ul`.

Reply:
>Rename multiple files using Perl regular expressions.
For the C-based command included in util-linux, see rename.ul.

This looks a bit strange to me. Changing the first sentence is fine but the `C-based` probably wont mean anything to most people.

Reply:
Yeah, I was unsure as well. In any case, the current version sounds good to me.